app/frontend/src/features/profile/edit/ProfileForm.tsx

```diff
@@ -199,17 +320,16 @@ export default function EditProfileForm() {
                 />
               )}
             />
-
-            <Box className={classes.buttonContainer}>
+            <div className={classes.buttonContainer}>
               <Button
                 type="submit"
                 variant="contained"
                 color="primary"
                 onClick={onSubmit}
               >
-                Save
+                {SAVE}
               </Button>
```

Comment:
This is stuck right against the bottom of the screen on the latest preview unlike in your screenshot. 🤔  
We probably want a bit of spacing below this?